Example/APMAlertController/ViewController.swift

```diff
@@ -194,25 +185,55 @@ class ViewController: UIViewController {
     func alertTextTitleAttributedMessage(_ indexPath: IndexPath) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/b9c4a952162e21ef3d6579eda097becf88e6343a/Example/APMAlertController/ViewController.swift#L185"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 44 lines</td>
    </tr>
  </tbody>
</table>